qa/suites/rgw/verify/tasks/rgw_tempest.yaml

```diff
@@ -0,0 +1,74 @@
+tasks:
+- install:
+    flavor: notcmalloc
+- ceph:
+- tox:
+  - client.0
+- keystone:
+    client.0:
+      sha1: 12.0.0.0b2
+      force-branch: master
+      tenants:
+        - name: admin
+          description:  Admin Tenant
+      users:
+        - name: admin
+          password: ADMIN
+          project: admin
+      roles: [ name: admin, name: Member ]
+      role-mappings:
+        - name: admin
+          user: admin
+          project: admin
+      services:
+        - name: keystone
+          type: identity
+          description: Keystone Identity Service
+        - name: swift
+          type: object-store
+          description: Swift Service
+      endpoints:
+        - service: keystone
+          publicurl: http://127.0.1:$(public_port)s/v2.0
```

Comment:
`127.0.1` missing a 0

does this localhost ip mean we can't configure multiple radosgw instances (possible on different machines) to use this keystone instance?

we might want to preprocess these urls to add the actual hostname, similar to how `tasks/rgw.py` does with `ctx.rgw.role_endpoints` in `assign_ports()`. you could then expose it through `ctx.keystone.endpoint`, and the rgw task could do this:
```python
    rgw_cmd = ['radosgw']
    if ctx.keystone:
        rgw_cmd.extend(['--rgw_keystone_url', ctx.keystone.endpoint])
```